### PR TITLE
Signal Management

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?
Adds the initial doc for Signal Management

https://docs-staging.datadoghq.com/kaylyn/signal-management/security_platform/security_signal_management/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/security_signal_management/_index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+further_reading:
+- link: "/cloud_siem/default_rules"
+  tag: "Documentation"
+  text: "Explore default detection rules"
+- link: "https://www.datadoghq.com/blog/detect-abuse-of-functionality-with-datadog/"
+  tag: "Blog"
+  text: "Detect Abuse of Functionality with Datadog"
+- link: "https://www.datadoghq.com/blog/impossible-travel-detection-rules/"
+  tag: "Blog"
+  text: "Detect suspicious login activity with impossible travel detection rules"
```
Comment (apigirl) [inline content/en/security_platform/security_signal_management/_index.md:13]:
```suggestion
```

File: content/en/security_platform/security_signal_management/_index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* Use the following search syntax @evt.name:"Security Monitoring" 
+* Select the "Security Monitoring" facet under the "Event Name" facet.
+
+
+[1]: /account_management/rbac/?tab=datadogapplication#pagetitle
+[2]: /account_management/rbac/permissions/#cloud-security-platform
+[3]: /security_platform/notification_rules/
+[4]: /account_management/audit_logs/#overview
+[5]: /account_management/audit_logs/#retention
+{{< partial name="whats-next/whats-next.html" >}}
```
Comment (maycmlee) [inline content/en/security_platform/security_signal_management/_index.md:64]:
```suggestion
```

Comment (maycmlee) [reply content/en/security_platform/security_signal_management/_index.md:64]: Would remove because there's no Further Reading section.

Comment (maycmlee) [reply content/en/security_platform/security_signal_management/_index.md:64]: Would remove because there's no Further Reading section.

**Review by maycmlee — APPROVED** [review]
Hi, just one small edit!

**Pull request merged**